### PR TITLE
docs: reference: commandline: daemon: fedora 23+ has mapping files

### PR DESCRIPTION
**Pull request by runcom**
as per https://bugzilla.redhat.com/show_bug.cgi?id=1309425
this is coming

Signed-off-by: Antonio Murdaca runcom@redhat.com

### CODE REVIEW

File: docs/reference/commandline/daemon.md [hunk 1]
```diff
@@ -715,10 +715,9 @@ the first 65536 range:
      dockremap:165536:65536
 ```
 
-> **Note:** On a fresh Fedora install, we had to `touch` the
-> `/etc/subuid` and `/etc/subgid` files to have ranges assigned when users
-> were created.  Once these files existed, range assignment on user creation
-> worked properly.
+> **Note:** On Fedora < 23, you have to `touch` the `/etc/subuid` and `/etc/subgid`
+> files to have ranges assigned when users were created.  Once these files
```
Comment (thaJeztah) [inline docs/reference/commandline/daemon.md]: when users were -> when users are

File: docs/reference/commandline/daemon.md [hunk 2]
```diff
@@ -715,10 +715,9 @@ the first 65536 range:
...
+> existed, range assignment on user creation worked properly.
```
Comment (thaJeztah) [inline docs/reference/commandline/daemon.md]: existed -> exist

worked -> work

File: docs/reference/commandline/daemon.md [hunk 3]
```diff
@@ -715,10 +715,9 @@ the first 65536 range:
      dockremap:165536:65536
 ```
 
-> **Note:** On a fresh Fedora install, we had to `touch` the
-> `/etc/subuid` and `/etc/subgid` files to have ranges assigned when users
-> were created.  Once these files existed, range assignment on user creation
-> worked properly.
+> **Note:** On Fedora < 23, you have to `touch` the `/etc/subuid` and `/etc/subgid`
```
Comment (thaJeztah) [inline docs/reference/commandline/daemon.md]: Perhaps "On Fedora 22 and older" (or just "Fedora 22", because we no longer build for "Fedora 21"

**Comment by runcom** [comment]
@thaJeztah updated PTAL

**Comment by thaJeztah** [comment]
LGTM!

ping @vdemeester for the merge :rabbit:

**Comment by vdemeester** [comment]
yay :tada: 
LGTM 🐰

**Comment by thaJeztah** [comment]
retest this please

**Pull request merged**